### DISCUSSION TITLE
Antialiased smooth round rect but with different colors in/out-side

### DISCUSSION THREAD

**Discussion by rradomir** [Bodmer/TFT_eSPI#2727] [General]
Hi, this can  be considered as proposition to be done in future. So I wanted to create sort of buttons, with round frame and background. If I select one of buttons, there should be bolder frame and I achieved this by `drawSmoothRoundRect`, but I can only select one background. 
No specified background (it is white):
![356384681_1446003309483937_3281934583199201803_n](https://github.com/Bodmer/TFT_eSPI/assets/97620566/8182fb84-db55-465e-b4eb-3ddfb064e0e1)

Black background:
![355619039_1246407845864295_8346751768517959090_n](https://github.com/Bodmer/TFT_eSPI/assets/97620566/16ed849a-e73f-44ee-be8a-8c5f3184c7c4)
So I think it would be nice to define inner and outer background to antialias :) or to turn off aliasing?

**Comment by Bodmer** [comment]
I will add to the rather long TODO list.

Adding a small background gap between the inner and outer zones is a potential workaround. I used this approach for meters:
![20230715_114549](https://github.com/Bodmer/TFT_eSPI/assets/15824805/bff51204-26d1-41e6-ad08-e7f666baed8a)

**Reply by Avamander** [reply]
Maybe this workaround could also be added to TFT_eWidget? Currently drawing buttons on top of a dynamic background will use antialiasing with a static color. It reading the display colors would be nice, but the workaround would likely be sufficient.
